src/test/java/andersen/AndersenTasks/collections/CustomArrayListTests.java

```diff
@@ -0,0 +1,55 @@
+package andersen.AndersenTasks.collections;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+public class CustomArrayListTests {
+    private CustomArrayList customArrayList;
+
+    @BeforeEach
+    public void init(){
+        customArrayList = new CustomArrayList<>();
+    }
+
+    @Test
+    public void testNotNull(){
+        Assertions.assertNotNull(customArrayList);
+    }
+
+    @Test
+    public void testNull(){
```

Comment:
For this one name should be something like defaultListSizeIsEqualsToZeroTest
by conventions it is recommended to add Test at the end, not at beginning